### DISCUSSION TITLE
Tutorial for save/load

### DISCUSSION THREAD

**Discussion by Snoopy137** [AlmasB/FXGL#1277] [Q&A]
Is there a video tutorial showing how to save and load a game?

**Comment by AlmasB** [comment]
No video tutorial, but there is a minimal sample showing how to use the SaveLoadService: [file](https://github.com/AlmasB/FXGL/blob/22c7ecf98ac2420afac8bd00b8cbed21f09b6f27/fxgl-samples/src/main/java/advanced/SaveLoadSample.java)
